bigtable/google/cloud/bigtable_v2/gapic/bigtable_client.py

```diff
@@ -584,15 +584,19 @@ def check_and_mutate_row(
                 ``predicate_filter`` yields at least one cell when applied to
                 ``row_key``. Entries are applied in order, meaning that earlier
                 mutations can be masked by later ones. Must contain at least one entry
-                if ``false_mutations`` is empty, and at most 100000.
+                if ``false_mutations`` is empty, and at most
+
+                100000.
```

Comment:
the extra line seems odd, should I add a replace to `synth.py`?

Reply:
Or figure out why the code generation is adding it, since there is no corresponding change in the `.proto` file.